public/eventsform.html

```diff
@@ -0,0 +1,106 @@
+<!DOCTYPE html>
+<html>
+<head>
+<meta name="viewport" content="width=device-width, initial-scale=1">
+<style>
+body {font-family: Arial, Helvetica, sans-serif;}
+* {box-sizing: border-box;}
+
+.open-button {
+  background-color: #555;
+  color: white;
+  padding: 16px 20px;
+  border: none;
+  cursor: pointer;
+  opacity: 0.8;
+  position: fixed;
+  bottom: 23px;
+  right: 28px;
+  width: 280px;
+}
+
+.form-popup {
+  display: none;
+  position: fixed;
+  bottom: 0;
+  right: 15px;
+  border: 3px solid #f1f1f1;
+  z-index: 9;
+}
+
+.form-container {
+  max-width: 300px;
+  padding: 10px;
+  background-color: white;
+}
+
+.form-container input[type=text], .form-container input[type=password] {
+  width: 100%;
+  padding: 15px;
+  margin: 5px 0 22px 0;
+  border: none;
+  background: #f1f1f1;
+}
+
+
+.form-container .btn {
+  background-color: #04AA6D;
+  color: white;
+  padding: 16px 20px;
+  border: none;
+  cursor: pointer;
+  width: 100%;
+  margin-bottom:10px;
+  opacity: 0.8;
+}
+
+.form-container .cancel {
+  background-color: red;
+}
+
+.form-container .btn:hover, .open-button:hover {
+  opacity: 1;
+}
+</style>
+</head>
+<body>
+
+<button class="open-button" onclick="openForm()">Add An Event</button>
+
+<div class="form-popup" id="myForm">
+  <form action="/action_page.php" class="form-container" method="get">
+    <h1>Add Event</h1>
+
+    <label for="Eventname"><b>Event Name</b></label>
+    <input type="text" placeholder="Event Name" name="Eventname" required>
+    
+    <label for="Event Time"><b>Event Time</b></label>
+    <input type="date" placeholder="Event Date" name="EventTime" required>
+    <br>
+    <br>
+    <label for="Event Gallery"><b>Event Gallery</b></label>
+    <input type="file" placeholder="Event Gallery" name="EventGallery" required>
```

Comment:
It does not take multiple images now
Line 82 should be replaced by this -
<input type="file" placeholder="Event Gallery" name="EventGallery" required multiple>
Now it will accept multiple images